src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -75,22 +79,33 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, Htt
             }
         }
 
-        public string GetRandomString(int maxLength)
+        /// Gets a random ASCII string within specified length range
+        public string GetRandomString(int minLength, int maxLength, bool alphaNumericOnly = true)
         {
-            int length = _random.Next(0, maxLength);
+            int length = _random.Next(minLength, maxLength);
             var sb = new StringBuilder(length);
             for (int i = 0; i < length; i++)
             {
-                sb.Append((char)(_random.Next(0, 26) + 'a'));
+                if (alphaNumericOnly)
+                {
+                    // alpha character
+                    sb.Append(alphaNumeric[_random.Next(alphaNumeric.Length)]);
```

Comment:
If this branch is intended to be a "valid header" counterpart to the `else`, consider making `alphaNumeric` consist of all the possible chars headers can have.

Reply:
It's mostly intented to cover existing usecases of the `GetRandomString()` case that assume output is alphanumeric. I use url encodings of ascii for random headers.